pkg/asset/installconfig/gcp/validation.go

```diff
@@ -63,6 +66,25 @@ func validateSubnet(client API, ic *types.InstallConfig, fieldPath *field.Path,
 	return nil
 }
 
+func validateClusterName(clusterName string) field.ErrorList {
```

Comment:
We have validation code in 2 places
- pkg types : this is about syntax and static validation ..
-asset : this is more about API based..

So I think we should have it in pkg types

Reply:
> So I think we should have it in pkg types

Yes I think that would be cleaner. Not sure why I put it in assets to begin with. Will move it over.